src/mol-gl/shader/shadows.frag.ts

```diff
@@ -80,19 +80,19 @@ float screenSpaceShadow(const in vec3 position, const in vec3 lightDirection, co
         rayCoords = uProjection * vec4(rayPos, 1.0);
         rayCoords.xyz = (rayCoords.xyz / rayCoords.w) * 0.5 + 0.5;
 
-        if (outsideBounds(rayCoords.xy))
-            return 1.0;
+        //if (outsideBounds(rayCoords.xy))
```

Comment:
are changes to this file needed for DoF?

Reply:
not need for dof, should if be a different PR ?

Reply:
removed it